homeassistant/components/media_player/braviatv.py

```diff
@@ -221,7 +221,8 @@ def __init__(self, host, mac, name, pin):
     def update(self):
         """Update TV info."""
         if not self._braviarc.is_connected():
-            self._braviarc.connect(self._pin, CLIENTID_PREFIX, NICKNAME)
+            if self._braviarc.get_power_status() != 'off':
```

Comment:
On my TV (Sony KDL-42W829B) it is possible, it does not need authentication.

I do not find reference for the HTTP API of Sony but it's seem logical that this kind of function do not need authentication for other model.